sdk/src/lib.rs

```diff
@@ -79,3 +84,11 @@ pub mod transport;
 extern crate serde_derive;
 pub extern crate bs58;
 extern crate log as logger;
+
+#[cfg(all(not(feature = "program")))]
```

Comment:
@jackcmay Hi! Just quick FYI. I'm going to add this conditional declaration to `sdk/src/lib.rs`. I hope this is harmless for the your daily life. :)

Also, I don't think we need to introduce `frozen_abi` for all of supported `cfg`s. So, I'm just excluding this from `feature = "program"`.
I presume ABI for a given `struct` should be universal across the Solana platform. Or is there any peculiar upcoming changes like adding `cfg(linux)` on individual `struct` fields in your mind? If that's the real, maybe I should be try hard to test the ABI under that `cfg` too. :)

Reply:
As long as stable-perf still builds we should be fine, just need to not break programs

Reply:
We will want to ABI enforce the interfaces that may be `program` specific so this isn't a good long term fix.

Reply:
Ok! Thanks for feedback! I'll consider rectifying this, then. :) As for `stable-perf`, it is unstable at the moment, unrelated to this PR.

Reply:
@jackcmay I succeeded to remove this `cfg` now. :)